widget/src/utils/env.ts

```diff
@@ -30,3 +30,5 @@ export const registerUriPathName = process.env.CLIENT_REGISTER_URI_PATH_NAME ||
 export const loginUriPathName = process.env.CLIENT_LOGIN_URI_PATH_NAME || "";
 
 export const forgotPasswordUriPathName = process.env.CLIENT_FORGOT_PASSWORD_URI_PATH_NAME || "";
+
+export const forceRedirect = process.env.FORCE_REDIRECT || false;
```

Comment:
just checking, did you check when the value set to FORCE_REDIRECT=false? it usually become string and when you check if(forceRedirect) it will treat it as true.

usually we do something like this
export const forceRedirect = process.env.FORCE_REDIRECT==="true" || false;

but I could be wrong, could you please check if it become string or not

Reply:
updated and tested for no assigned, false and true value